src/dune

```diff
@@ -1,4 +1,3 @@
 (library
  (public_name cmdliner)
- (flags :standard -w -3-6-27-32-35)
```

Comment:
And that won't do it, or will it ?

Reply:
why? By default in dune `(flags :standard)` and the `(env ...)` stanza above simply changes the value of `:standard` for everything is the project. So it just works fine.